app/src/main/java/com/keylesspalace/tusky/components/search/fragments/SearchStatusesFragment.kt

```diff
@@ -173,9 +172,17 @@ class SearchStatusesFragment : SearchFragment<StatusViewData.Concrete>(), Status
         }
     }
 
-    override fun onFavourite(favourite: Boolean, position: Int) {
-        adapter?.peek(position)?.let { status ->
-            viewModel.favorite(status, favourite)
+    override fun onFavourite(favourite: Boolean, position: Int, animationCallback: () -> Unit) {
+        val status = adapter?.peek(position)?.asStatusOrNull() ?: return
+
+        if (favourite) {
+            confirmReblog(preferences) { visibility ->
```

Comment:
This should be `confirmFavorite`, right?

Reply:
yes, thank you!